pkg/xray/watches.go

```diff
@@ -0,0 +1,246 @@
+package xray
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"log"
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type WatchGeneralData struct {
+	Name        string `json:"name"`
+	Description string `json:"description,omitempty"`
```

Comment:
Same comment applies to `omitempty`. Please remove.

Reply:
removed